src/server.js

```diff
@@ -0,0 +1,40 @@
+const FeedParser = require('feedparser')
+const request = require('request')
+const Rx = require('rx') 
+
+// Event loop will run 3 times for testing
+const interval = Rx.Observable.interval(5000).timeInterval().take(3)
```

Comment:
Will do, there's going to be a few more configuration bits to this so I'm glad the common package handles it.